src/middleware/index.js

```diff
@@ -69,31 +70,31 @@ const analyticsMiddleware = store => next => action => {
             timestamp: new Date().toString(),
             account_id: wallet.accountId
         })
+        let id = mixpanel.get_distinct_id();
+        mixpanel.identify(id);
         mixpanel.people.set({
             network_id: networkId,
             stake: 0
         })
         if (trackingEvents.includes(action.type)){
             mixpanel.track(action.type, details);
         }
-        if (action.type === 'CREATE_NEW_ACCOUNT'){
```

Comment:
Following changes don't seem to match PR topic of __fix identify__.

So either PR description should be updated or these changes should be in the separate PR with separate explanation.

Reply:
it is part of setting user profile and I will change the description.